index.js

```diff
@@ -26,5 +26,14 @@ export default async function gitLog (options = {}) {
 
   args.push(options.range || 'HEAD')
 
+  if (options?.path) {
+    args.push("--")
+    const paths = Array.isArray(options.path) ? options.path : [options.path]
+
+    paths.forEach((path) => {
+      args.push(path)
+    })
```

Comment:
`push` accepts multiple values, so instead of looping we can just do one call. Something like this maybe:

```suggestion

    if (Array.isArray(options.path)) {
      args.push(...options.path)
    } else {
      args.push(options.path)
    }
```